content/applications/websites/livechat/chatbots.rst

```diff
@@ -0,0 +1,214 @@
+========
+Chatbots
+========
+
+A *chatbot* is a program designed to mimic a conversation with a live human. Chatbots are assigned
+a script of pre-written steps to follow. These scripts are designed to anticipate a visitor's
+potential response, and lead them through a series of questions and answers the same way an live
+team member would. Chatbots can be customized to fill various roles, depending on the page of the
```

Comment:
Maybe you could name a few of the various roles here that a chatbot could help fill? Unless that's better left unsaid and just explained in depth below, up to you

Reply:
I originally had included a list of examples later in the doc, but I removed them because I was worried it looked like they would either be pre-configured with the app or that it was starting to look a bit too "sales-y".  However, I don't think it would be an issue to include them in the intro as long as I don't expand too much. 👍